.github/workflows/CI_agent_sshlibrary.yml

```diff
@@ -0,0 +1,41 @@
+name: Publish docker sshlibrary agent image on dockerhub
+on:
+  push:
+
```

Comment:
Consider adding something like this to `push`:
```
    paths:
      - 'agent/rfswarm-sshlibrary/*'
```
See: [Running your workflow only when a push affects specific files](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-only-when-a-push-affects-specific-files)

You'll probably want something similar for the base image

Reply:
Hello Dave, take  for your reply. .K I'm having a look now. Ksenia made some big changes by adding tag name versioning and  build images only when changes are detected. So I'm going to check all of them at the same time.

Reply:
Ok I just checked and Ksenia has already took your suggestion in account :-) . Thanks Dave